Index: karaage/docs/conf.py
===================================================================
--- karaage.orig/docs/conf.py	2014-09-19 14:30:53.891322919 +1000
+++ karaage/docs/conf.py	2014-09-19 14:42:46.542363188 +1000
@@ -29,6 +29,6 @@
 extensions += ["sphinx.ext.intersphinx"]
 intersphinx_mapping = {
     'django': (
-        'https://docs.djangoproject.com/en/1.7/',
-        'https://docs.djangoproject.com/en/1.7/_objects/'),
+        '/usr/share/doc/python-django-doc/html/',
+        None),
 }
